cluster/environments/azure-single-keyvault/README.md

```diff
@@ -7,3 +7,121 @@ The `azure-single-keyvault` environment deploys a single production level AKS cl
 1. Copy this template directory to a repo of its own. Bedrock environments remotely reference the Terraform modules that they need and do not need be housed in the Bedrock repo.
 2. Follow the instructions on the [main Azure page](../../azure) in this repo to create your cluster and surrounding infrastructure.
 
+## Deploy the Environment
+
+The `azure-single-keyvault` uses the `backend.tfvars` and requires that you create another .tfvars if it does not already exists (e.g. `terraform.tfvars`).
+
+`backend.tfvars` (**NOTE**: you can and should use the same `backend.tfvars` that was used to deploy `azure-common-infra`, but with a different key as shown below):
+
+```
+storage_account_name="myStorageAccount"
+
+access_key="gmnNFIa/LyKgbC5CZn9Io8jbngSW+Isa5vTZfKsIgkz/8EG2YPn4CV42hMDXmQ73zDu5Y7puFkAoWPHLtE6/w=="
```

Comment:
Is this a real access key, do we wanna remove it?

Reply:
It's not a real access key, I modified it. But will remove it! Thanks!